src/core_modules/capture-core/components/WidgetFeedback/WidgetFeedbackContent/WidgetFeedbackContent.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+import React, { type ComponentType } from 'react';
+import { spacers, spacersNum, colors } from '@dhis2/ui';
+import i18n from '@dhis2/d2-i18n';
+import { withStyles } from '@material-ui/core';
+import type { filteredKeyValue, filteredText, contentType } from '../WidgetFeedback.types';
+
+const styles = {
+    container: {
+        padding: `0px ${spacersNum.dp16}px`,
+    },
+    unorderedList: {
+        paddingLeft: spacersNum.dp16,
+        marginTop: '0px',
+        lineHeight: '1.375',
+        fontSize: spacers.dp16,
+        color: colors.grey900,
+    },
+    noFeedbackText: {
+        color: colors.grey600,
+        paddingLeft: spacersNum.dp16,
+        fontWeight: 400,
+        fontSize: spacers.dp16,
+        marginTop: '0px',
+    },
+    listItem: {
+        marginBottom: spacersNum.dp4,
+        '&::marker': {
+            color: colors.grey500,
+        },
+    },
+};
+
+
+const WidgetFeedbackContentComponent = ({ widgetData, classes }: contentType) => {
+    if (!widgetData?.length) {
+        return <p data-test={'widget-content'} className={classes.noFeedbackText}>{i18n.t('No feedback for this enrollment yet')}</p>;
```

Comment:
I think we must pass in this emptyText to the component (will be another text when we're looking at one specific event)